iceoryx_posh/include/iceoryx_posh/internal/popo/building_blocks/chunk_distributor.hpp

```diff
@@ -124,7 +124,7 @@ class ChunkDistributor
     MemberType_t* getMembers() noexcept;
 
   private:
-    MemberType_t* m_chunkDistrubutorDataPtr;
+    MemberType_t* m_chunkDistrubutorDataPtr{nullptr};
```

Comment:
There is only on c'tor which sets this pointer (with cxx::not_null). When I see this change I could get the impression that this member could also be nullptr. I also already saw QAC warnings that complained when you set a member in the initializer list and with this initializer. So I decided to not have this nullptr initialization. But maybe this is a matter of taste..

Reply:
Here I have the C++ Core Guidelines on my side and I think the CTor approach is actually a bad code style: https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rc-in-class-initializer 

In this particular case I would suggest to ignore the QAC warning and perform all the initialization inside the header and only specialize the member variables during the constructor initialization.